README.md

```diff
@@ -33,6 +33,9 @@ You will need the following to build the latest Dynamo on Windows:
 - [Microsoft Visual Studio 2019](https://visualstudio.microsoft.com/downloads/) (any edition)
 - Microsoft .NET Framework 4.8 (included with Visual Studio 2019)
 - Node.js v16 and npm v8
+- - [Download](https://nodejs.org/en/download/) and install the latest LTS version of Node.js
+- - Open up your Command Prompt and check you have successfully installed Node by inputting `node -v`. If you have Node installed, it will tell you the version.
+- - In the Command Prompt, install Node Package Manager (NPM) by typing `npm install` and checking you have correctly installed it with `npm -v`
```

Comment:
this step is not necessary - npm is installed when you run the node installer.
`npm install` will not install npm, it will install the packages specified by the package.json / package lock.

Reply:
agreed

Reply:
Ok cool - updating now!